jsettlers.mapcreator/src/jsettlers/exceptionhandler/ErrorIcon.java

```diff
@@ -0,0 +1,58 @@
+package jsettlers.exceptionhandler;
```

Comment:
I usually directly add it when I create a new file. However, I think Eclipse has a way to add this, or there is at least a plugin to do it automatically.